cmd/tle/commands/commands.go

```diff
@@ -156,7 +161,9 @@ func validateFlags(f *Flags) error {
 						"You might want to also specify a custom chainhash with the -c/--chain flag.\n\n")
 			}
 		}
-
+		break
+	case f.Metadata:
+		break
```

Comment:
Switch statements in Go only evaluate one case, so no need to `break`.

Also if the chain or network is the empty string there's something wrong since these have default values. 

```suggestion
	case f.Metadata:
		if f.Chain == "" {
			return fmt.Errorf("-c/--chain can't be the empty string")
		}
		if f.Network == "" {
			return fmt.Errorf("-n/--network can't be the empty string")
		}
```